packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -1,16 +1,40 @@
-part of '../../neon.dart';
+part of 'blocs.dart';
 
 const _keyAccounts = 'accounts';
 
-abstract class AccountsBlocEvents {
+abstract interface class AccountsBlocEvents {
+  /// Logs in the given [account].
+  ///
+  /// It will also call [setActiveAccount] hen no other accounts are registered in [AccountsBlocStates.accounts].
   void addAccount(final Account account);
+
+  /// Logs out the given [account].
+  ///
+  /// If [account] is the current [AccountsBlocStates.activeAccount] it will automativcally activate the first one in [AccountsBlocStates.accounts].
```

Comment:
```suggestion
  /// If [account] is the current [AccountsBlocStates.activeAccount] it will automatically activate the first one in [AccountsBlocStates.accounts].
```